WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReaders/CardReaderSettingsViewModel.swift

```diff
@@ -162,10 +162,19 @@ final class CardReaderSettingsViewModel: ObservableObject {
 
     /// Views can call this method to disconnect from the connected reader.
     func disconnectAndForget() {
-        // TODO dispatch an action to disconnect.
-        connectedReader = nil
-        activeView = .connectYourReader
-        activeAlert = .none
+        let action = CardPresentPaymentAction.disconnect { [weak self] result in
```

Comment:
This UI is temporary and this view model may or may not be thrown away soon. 

Based on that, I didn't add any user facing errors. If this view model is meant to remain then let's log a new ticket to present an error here, in the unlike event of a failure when disconnecting.